### DISCUSSION TITLE
ESP32-S3 & ILI9488 Pinout and bin

### DISCUSSION THREAD

**Discussion by maix-me** [HASwitchPlate/openHASP#678] [Q&A]
Hi everyone, how how to connect this display with this board?
i need bin firmware and piount to connect
I try to connet with this manual https://www.openhasp.com/0.7.0/hardware/generic/ili9341-ips/ but i did not hav GPIO23, GPIO26 and N/C

**Comment by lpacchin** [comment]
Hi
same combination as mine, you can try my compiled bin: 

[esp32-s3-ili9488_full_16MB_v0.7.0-rc11_4b6a002.bin](https://github.com/lpacchin/lp_openhasp/blob/main/esp32-s3-ili9488_full_16MB_v0.7.0-rc11_4b6a002.bin
)

openHASP for:

esp32-s3-devkitc-1-n16r8

Pinout:

TFT_CS=10 
TFT_DC=7  
TFT_RST=6 
TFT_BCKL=5 
TFT_MOSI=11   Connect T_DIN of touch and SDI of TFT
TFT_SCLK=12   Connect T_CLK of touch and SCK of TFT
TFT_MISO=13   Connect T_D0 of touch (only)
TOUCH_CS=4

**Reply by maix-me** [reply]
Wich software do you use to flash this bin file?
I try tasmotizer and it`s send me error: Invalid head of packet
And try Tasmota-PyFlasher-1.0 and it`s no sucsess with no error

**Comment by lpacchin** [comment]
I use https://web.esphome.io/

**Reply by maix-me** [reply]
it`s installed. But i`m not configure wifi. Witch password to here AP ?

**Reply by lpacchin** [reply]
https://www.openhasp.com/0.7.0/0.7.0/firmware/wifi-setup/
haspadmin

**Reply by maix-me** [reply]
You only this pin connect?
Pinout:

TFT_CS=10
TFT_DC=7
TFT_RST=6
TFT_BCKL=5
TFT_MOSI=11 Connect T_DIN of touch and SDI of TFT
TFT_SCLK=12 Connect T_CLK of touch and SCK of TFT
TFT_MISO=13 Connect T_D0 of touch (only)
TOUCH_CS=4

**Comment by lpacchin** [comment]
If do not see try to connect the TFT_BCKL to +3v maybe the backlight is too low
Also make sure you connect 3v3 and ground

**Reply by maix-me** [reply]
Thank you!

**Reply by T-Bone-one** [reply]
> If do not see try to connect the TFT_BCKL to +3v maybe the backlight is too low Also make sure you connect 3v3 and ground

Hi . i loaded your bin file. Everything is working except the onboard RGB Led on Pin48 ... is this missing in your bin file ?
Thank you
